todo_app/app.py

```diff
@@ -1,34 +1,45 @@
 from flask import Flask, render_template, request
-from todo_app.flask_config import Config
-from todo_app.data.session_items import *
+from .data.todoapi import TrelloAPI
+from dateutil import parser
 
 app = Flask(__name__)
-app.config.from_object(Config)
+todoapi = TrelloAPI()
 
 def render_index_response():
-    return render_template('index.html', items = get_items())
+    return render_template('index.html', 
+        items_list = sorted(todoapi.get_list_of_items(), 
+        key=lambda item : (item.status, item.id)), 
+        statuses = todoapi.board.statuses, 
+        todomapper = todoapi.todomapper, 
+        board = todoapi.board.name
+    )
 
 @app.route('/', methods=['GET'])
 def index():
     return render_index_response()    
 
 @app.route('/add', methods=['POST'])
 def add():
-    add_item(request.form.get('title'))
+    todoapi.add_item(request.form)
     return render_index_response()
```

Comment:
This should `return redirect(url_for('index'))`

The difference is the server will send a redirect response to the browser which will then make a new request to the index page. The main benefits are: URL displayed in the browser address bar will be correct, refreshing the page won't resubmit the form.

Reply:
Doing so returns 306. The intention heer is not do /add at the top but rather have single page